packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php

```diff
@@ -125,76 +131,77 @@
             <div class="header">
                 <div class="row">
                     <div class="col-12">
+                        @if (core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo'))
+                            <div class="image text-center">
+                                <img class="logo" src="{{ Storage::url(core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo')) }}" alt=""/>
+                            </div>
+                        @endif
                         <h1 class="text-center">{{ __('admin::app.sales.invoices.invoice') }}</h1>
                     </div>
                 </div>
+                <div class="row">
+                    <div class="col-12">
+                        <div class="col-6">
+                            <div class="merchant-details">
+                                <div>
+                                    <span class="merchant-details-title">{{ core()->getConfigData('sales.shipping.origin.store_name') ? core()->getConfigData('sales.shipping.origin.store_name') : '' }}</span>
+                                </div>
 
-                @if (core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo'))
-                    <div class="image">
-                        <img class="logo" src="{{ Storage::url(core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo')) }}" alt=""/>
-                    </div>
-                @endif
-
-                <div class="merchant-details">
-                    <div>
-                        <span class="merchant-details-title">{{ core()->getConfigData('sales.shipping.origin.store_name') ? core()->getConfigData('sales.shipping.origin.store_name') : '' }}</span>
-                    </div>
+                                <div>{{ core()->getConfigData('sales.shipping.origin.address1') ? core()->getConfigData('sales.shipping.origin.address1') : '' }}</div>
 
-                    <div>{{ core()->getConfigData('sales.shipping.origin.address1') ? core()->getConfigData('sales.shipping.origin.address1') : '' }}</div>
+                                <div>
+                                    <span>{{ core()->getConfigData('sales.shipping.origin.zipcode') ? core()->getConfigData('sales.shipping.origin.zipcode') : '' }}</span>
```

Comment:
Null coalescing operator can be used here



<br class="Apple-interchange-newline">